clubs/models.py

```diff
@@ -71,6 +61,115 @@ class Club(models.Model):
         null=True
     )
 
+    occupation = models.CharField(
+        help_text=("Club captain's occupation."),
+        blank=False,
+        null=True,
+        max_length=500
+    )
+
+    regional_coordinate = models.CharField(
+        help_text=("Regional Coordinator associated with club captain."),
+        blank=True,
+        null=True,
+        max_length=500
+    )
+
+    hosting_reason = models.TextField(
+        help_text=("Reason to host a Mozilla club."),
+        blank=False,
+        null=True
+    )
+
+    how_they_heard = models.CharField(
+        help_text="How did the applicant hear about Current approval status of the club.",
+        blank=False,
+        null=True,
+        max_length=100
+    )
+
+
+    '''
+        "Page 2" club-specific values
+    '''
+
+    name = models.CharField(
+        help_text="The name of the club.",
+        max_length=100
+    )
+
+    description = models.TextField(
+        help_text="Description of the club.",
+        blank=False,
+        null=False
+    )
+
+    venue = models.CharField(
+        help_text="Where does this club meet.",
+        blank=False,
+        null=True,
+        max_length=1000
+    )
+
+    # this is a range, and possibly "other" as user-supplied
+    # information. As such, this is a text field.
+    frequency = models.CharField(
+        help_text=("How frequently this club meets."),
+        blank=False,
+        null=True,
+        max_length=500
+    )
+
+    age_range = models.CharField(
+        help_text=("The club member age range."),
+        blank=False,
+        null=True,
+        max_length=500
+    )
+
+    club_size = models.CharField(
```

Comment:
not right now - it feels like this is a field that is super likely to get a tweak sooner than later, and I don't want to implicitly-link the teach-api and learning.moz codebases on this value quite yet.